docs/intro/tutorial.rst

```diff
@@ -1,141 +1,446 @@
+
+However, in this tutorial you will use a specific project directory structure
+designed with web-poet best practices in mind. Consider using the same project
+directory structure in all your web-poet projects.
+
+First create your project directory: ``tutorial-project/``.
+
+Within the ``tutorial-project`` directory, create:
+
+-   A ``run.py`` file, a file specific to this tutorial where you will put code
+    to test the execution of your web data extraction code.
+
+-   A ``tutorial`` directory, where you will place your web data extraction code.
+
+Within the ``tutorial-project/tutorial`` directory, create:
+
+-   An ``__init__.py`` file, so that the ``tutorial`` directory becomes an
+    importable Python module.
+
+-   An ``items.py`` file, where you will define item classes to store extracted
+    data.
+
+-   A ``pages`` directory, where you will define your page object classes.
+
+Within the ``tutorial-project/tutorial/pages`` directory, create:
+
+-   An ``__init__.py`` file.
 
+-   A ``books_toscrape_com.py`` file, for page object class code targeting
+    `books.toscrape.com`_.
 
-    class BookLinksPage(WebPage):
+Your project directory should look as follows:
 
-        @property
-        def links(self):
-            return self.css(".image_container a::attr(href)").getall()
+.. code-block:: text
 
-        def to_item(self) -> dict:
-            return {
-                "links": self.links,
-            }
+   tutorial-project
+   ├── run.py
+   └── tutorial
+       ├── __init__.py
+       ├── items.py
+       └── pages
+           ├── __init__.py
+           └── books_toscrape_com.py
 
-Downloading Response
+
+Create an item class
 ====================
 
-The ``BookLinksPage`` Page Object requires a
-:class:`~.HttpResponse` with the
-book list page content in order to extract the information we need. First,
-let's download the page using ``requests`` library.
+While it is possible to store the extracted data in a Python dictionary, it is
+a good practice to create an item class that:
+
+-   Defines the specific attributes that you aim to extract, triggering an
+    exception if you extract unintended attributes or fail to extract expected
+    attributes.
+
+-   Allows defining default values for some attributes.
+
+web-poet uses itemadapter_ for item class support, which means that any kind of
+item class can be used. In this tutorial, you will use attrs_ to define your
+item class.
+
+.. _attrs: https://www.attrs.org/en/stable/
+.. _itemadapter: https://github.com/scrapy/itemadapter
+
+Copy the following code into ``tutorial-project/tutorial/items.py``:
+
+.. literalinclude:: /tutorial-project/tutorial/items.py
+   :language: python
+   :lines: 1-6
+
+This code defines a ``Book`` item class, with a single required ``title``
+string attribute to store the book title.
+
+``Book`` is a minimal class designed specifically for this tutorial. In real
+web-poet projects, you will usually define item classes with many more
```

Comment:
Do you think if we can mention (as an example) and add a link to zyte-common-items here?